recipes/terraclimate/meta.yml

```diff
@@ -0,0 +1,26 @@
+title: "TerraClimate Monthly Surface Dataset"
+description: "High-resolution monthly climate and climatic water balance dataset"
+pangeo_forge_version: "0.4.0"
+pangeo_notebook_version: "2021.06.05"
+recipes:
+  - id: terraclimate
+    object: "recipe:recipe"
+provenance:
+  providers:
+    - name: "Climatology Lab"
+      description: "Climatology Lab at University of California Merced (http://www.climatologylab.org/)"
+      roles:
+        - producer
+        - licensor
+      url: http://www.climatologylab.org/terraclimate.html
+  license: "CC-0-1.0"
+maintainers:
+  - name: "Raphael Hagen"
+    orcid: "0000-0003-1994-1153"
+    github: norlandrhagen
+bakery:
+  id: "devseed.bakery.development.aws.us-west-2"
+  target: pangeo-forge-aws-bakery-flowcachebucketdasktest4-10neo67y7a924
+  resources:
+    memory: 4096
+    cpu: 1024
```

Comment:
This is great! It does bring up a question as to how recipe developers will choose a bakery though. As I understand it, there are two active bakeries (azure and aws), and all things being equal, I'd suggest we target the azure bakery on this one (this dataset is currently part of azure's public data program). But I've been out of the loop on this decision topic so please redirect me if there's already guidance on how this will work.

cc @TomAugspurger

Reply:
I don't think that the user should have to always specify resource requirements in meta.yaml. Can there be a default here?

Reply:
@rabernat I'll update `pangeo-forge-prefect` to support a default option.